tests/components/cloud/test_http_api.py

```diff
@@ -177,96 +237,143 @@ async def test_login_view_create_pipeline_fail(
     create_pipeline_mock.assert_awaited_once_with(hass, "cloud", "cloud")
 
 
-async def test_login_view_random_exception(cloud_client) -> None:
+async def test_login_view_random_exception(
+    cloud: MagicMock,
+    setup_cloud: None,
+    cloud_client: TestClient,
+) -> None:
     """Try logging in with invalid JSON."""
```

Comment:
Since we change the surrounding lines, let's fix the misleading docstring too

Reply:
Fixed.